cfgov/v1/tests/atomic_elements/organisms/test_organisms.py

```diff
@@ -216,3 +216,18 @@ def test_item_introduction(self):
         response = django_client.get('/learn/')
         self.assertContains(response, 'Item Introduction')
         self.assertContains(response, 'Item introduction body')
+
+    def test_html_block(self):
+        """ HTML Block correctly renders HTML on a Browse Page"""
+        browse_page = BrowsePage(
+            title='Browse Page',
+            slug='browse',
+        )
+        browse_page.content = StreamValue(
+            browse_page.content.stream_block,
+            [atomic.html_block],
+            True
+        )
+        publish_page(child=browse_page)
+        response = self.client.get('/browse/')
```

Comment:
Good catch Andy. I'm inclined to merge this for now but maybe can set up a test HTML page as part of the test.